README.md

```diff
@@ -157,15 +157,15 @@ spec:
           servicePort: 80
 ```
 
-You can also directly modify the service in your Tyk Dasboard, though if the ingress is recreated, the changes may not be retained.
+You can also directly modify the service in your Tyk Dashboard, though if the ingress is recreated, the changes may not be retained.
 
 When an ingress is removed, the services will be removed from the API Gateway as well.
 
 ### Templates
 
 It's quite likely that you will not want to overload your ingress specifications with annotations that set specific values in your API Definition. To make adding APIs much more flexible, you can make use of a single `template.service.tyk.io/` annotation to specify the name of a template to use when deploying your service to the gateway.
 
-This can be extremely useful if you want to standardise on certain types of service, e.g. "open-public", "closed-public" and "closed-jwt-internal", where you apply different auth scehemes, IP white lists and more complex re-usable specifications such as IDP provider details and secrets that you don;t want to re-code into each definition.
+This can be extremely useful if you want to standardize on certain types of service, e.g. "open-public", "closed-public" and "closed-jwt-internal", where you apply different auth schemes, IP white lists and more complex re-usable specifications such as IDP provider details and secrets that you don't want to re-code into each definition.
```

Comment:
```suggestion
This can be extremely useful if you want to standardise on certain types of service, e.g. "open-public", "closed-public" and "closed-jwt-internal", where you apply different auth schemes, IP white lists and more complex re-usable specifications such as IDP provider details and secrets that you don't want to re-code into each definition.
```